examples/dgmulti_3d/elixir_euler_tensorWedge.jl

```diff
@@ -0,0 +1,59 @@
+using OrdinaryDiffEq
+using Trixi, StartUpDG
```

Comment:
Do we need to load StartUpDG explicitly?

Reply:
No we don't, thanks for the hint.